pkg/ccl/backupccl/restore_planning.go

```diff
@@ -1447,37 +1447,44 @@ func checkClusterRegions(
 // that the manifests we are about to restore are from backups taken on a
 // version compatible with our current version.
 func checkBackupManifestVersionCompatability(
-	p sql.PlanHookState,
-	currentActiveVersion clusterversion.ClusterVersion,
+	ctx context.Context,
+	version clusterversion.Handle,
 	mainBackupManifests []backuppb.BackupManifest,
 	unsafeRestoreIncompatibleVersion bool,
 ) error {
 	// We support restoring a backup that was taken on a cluster with a cluster
 	// version >= the earliest binary version that we can interoperate with.
-	minimumRestoreableVersion := p.ExecCfg().Settings.Version.BinaryMinSupportedVersion()
+	minimumRestoreableVersion := version.BinaryMinSupportedVersion()
+	currentActiveVersion := version.ActiveVersion(ctx)
+
+	// Skip the version check if the user runs the restore with
```

Comment:
if for any reason you need another CI run, I might move this up to be the first thing the function does (before setting the local vars for versions)

Reply:
Done. I needed to update a test that was asserting the error message included the "restoreable" typo.